semgrep-core/src/runner/Run_semgrep.ml

```diff
@@ -358,15 +358,12 @@ let iter_targets_and_get_matches_and_exn_to_errors config f targets =
 (* File targeting and rule filtering *)
 (*****************************************************************************)
 
-let rules_for_xlang xlang rules =
```

Comment:
@aryx I saw in another PR you exposed this function. I deleted it since I didn't see it used anywhere else, but I can put it back

Reply:
yes please, put it back, it's used in deep-semgrep
(and it could be useful also I think when we run semgrep-core without -targets but with simply -lang)